contrib/upldrel/entrypoint.sh

```diff
@@ -17,9 +17,11 @@ gcloud_init "$GCPJSON_FILEPATH"
 
 echo "Uploading archive as $TO_FILENAME"
 gsutil cp "$FROM_FILEPATH" "gs://$BUCKET/$TO_FILENAME"
-gsutil cp "$FROM_FILEPATH" "gs://$BUCKET/$ALSO_FILENAME"
+[[ -z "$ALSO_FILENAME" ]] || \
+    gsutil cp "$FROM_FILEPATH" "gs://$BUCKET/$ALSO_FILENAME"
 
 echo "."
 echo "Release now available for download at:"
-echo "    https://storage.cloud.google.com/$BUCKET/$TO_FILENAME"
-echo "    https://storage.cloud.google.com/$BUCKET/$ALSO_FILENAME"
+echo "    https://storage.googleapis.com/$BUCKET/$TO_FILENAME"
+[[ -z "$ALSO_FILENAME" ]] || \
+    echo "    https://storage.googleapis.com/$BUCKET/$ALSO_FILENAME"
```

Comment:
did you mean to add this file?  If so, could you update the description and possibly the title of the PR?

Reply:
Yes, I added a new commit for it but didn't update the PR, I'll do that now.  Thanks for the reminder.